typescript/infra/config/environments/mainnet3/warp/configGetters/getEclipseEthereumUSDTWarpConfig.ts

```diff
@@ -0,0 +1,37 @@
+import { ethers } from 'ethers';
+
+import { ChainMap, TokenRouterConfig, TokenType } from '@hyperlane-xyz/sdk';
+
+import { tokens } from '../../../../../src/config/warp.js';
+import { getRegistry as getMainnet3Registry } from '../../chains.js';
+import { DEPLOYER } from '../../owners.js';
+
+export const getEclipseEthereumUSDTWarpConfig = async (): Promise<
+  ChainMap<TokenRouterConfig>
+> => {
+  const registry = await getMainnet3Registry();
+
+  // @ts-ignore - foreignDeployment configs don't conform to the TokenRouterConfig
+  const eclipsemainnet: TokenRouterConfig = {
+    type: TokenType.synthetic,
+    foreignDeployment: '5g5ujyYUNvdydwyDVCpZwPpgYRqH5RYJRi156cxyE3me',
+    gas: 300_000,
+    mailbox: (await registry.getChainAddresses('eclipsemainnet'))!.mailbox,
+    owner: '9bRSUPjfS3xS6n5EfkJzHFTRDa4AHLda8BU2pP4HoWnf',
+    interchainSecurityModule: ethers.constants.AddressZero,
+  };
+  let ethereum: TokenRouterConfig = {
+    isNft: false,
+    type: TokenType.collateral,
+    token: tokens.ethereum.USDT,
+    owner: DEPLOYER,
+    gas: 300_000,
+    mailbox: (await registry.getChainAddresses('ethereum'))!.mailbox,
```

Comment:
instead of doing this imo we should use the router config like we do here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumEclipseUSDCWarpConfig.ts#L14

Reply:
initially wanted to, but the config generation scripts (e.g. `generate-eclipse-usdt-warp-route-config.ts`) don't have the router config and instantiating it seemed a pain

Reply:
the pzETH / ezETH configs didn't take the router as an arg for similar reasons, I believe

Reply:
should be able to get it with somethign like https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/db9196837296c55f94f32680f33121505e696641/typescript/infra/scripts/deploy.ts#L220-L221